modules/gitops-configuring-namespace-management-argocd-tenants.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * multitenancy/multitenancy-support-in-gitops.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="configuring-namespace-management-argocd-tenants_{context}"]
+= Configuring namespace management for Argo CD tenants
+
+You can enable non-cluster-admin users to configure namespace management for their Argo CD instances by creating a `NamespaceManagement` custom resource (CR).
+
+.Prerequisites
+
+* You have logged in to the {OCP} cluster as an administrator.
+* You have installed the {gitops-shortname} Operator on your {OCP} cluster.
+* You can access the default Argo CD instance in the `openshift-gitops` namespace.  
+
+.Procedure
+
+. Enable the feature in the `Subscription` Custom Resource (CR).
++
+Edit the {gitops-title} Operator `Subscription` CR to allow namespace management.  
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: argocd-operator
+  namespace: argocd
+spec:
+  config:
+    env:
+    - name: ALLOW_NAMESPACE_MANAGEMENT_IN_NAMESPACE_SCOPED_INSTANCES
+      value: "true" # <1>
+----
+<1> Specify the value to `true` to enable namespace management in the `Subscription` CR
+
+. Configure the Argo CD CR.  
++
+Update the Argo CD instance to allow namespace management.  
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  namespace: <argocd-ns> # <1>
+spec:
+  namespaceManagement:
+    - name: <tenant-ns> # <2>
+      allowManagedBy: true
+----
++
+<1> Replace `<argocd-ns>` with the Argo CD instance namespace.  
+<2> Replace `<tenant-ns>` with the namespace you want Argo CD to manage.
+
+. Create a `NamespaceManagement` CR.  
++
+In the tenant namespace, the tenant user creates a `NamespaceManagement` CR:  
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: NamespaceManagement
+metadata:
+  name: ui-team-namespace
+  namespace: <tenant-ns> # <1>
+spec:
+  managedBy: <argocd-ns> # <2>
+----
++
+<1> `.metadata.namespace` - Namespace to be managed by the Argo CD instance.  
+<2> `.spec.managedBy` - Namespace where the Argo CD instance runs.  
```

Comment:
Could add a second sentence like: 
`The namespace name specified here must exactly match the Argo CD namespace.`

(Mainly this sentence is a simpler way of saying that this `mangedBy` field is NOT a glob, in contrast to the Argo CD field above which IS a glob. I can see users getting confused and thinking they both work the same way. This second sentence helps clarify that they do not.)

Reply:
Fixed.